lib/testing/src/lib/protractor/process-services-cloud/pages/form/widget/attach-file-widget-cloud.page.ts

```diff
@@ -63,6 +63,12 @@ export class AttachFileWidgetCloudPage {
         }
     }
 
+    async checkNoFileIsAttached(): Promise<void> {
```

Comment:
Are you sure we need this? Why a new method? My impression is we need to update this page https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L44-L48
and also the ADW testing pages with the new locators

Reply:
its required in APA, to update failing tests

Reply:
there is a condition to check for no files, so this method is required

Reply:
the cloud attach-widget page was updated while merging attach-file-widget changes only. Only I have to update the ADW APA which I'm creating a new PR in apps, and this function was missing from attach-widget-cloud page so added it

Reply:
If there are failing tests in APA due to a change, it is expected to update an existing method. Why are we creating a new one? The method that you are creating exists in this page https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L44-L48

Reply:
And this one as well, is being used in a failing test from APA. We need to update the locator https://github.com/Alfresco/alfresco-ng2-components/blob/d3863077cf0b18d6f210f987f27d13fe3358aa3e/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L168

Reply:
Yes, but we are not using `AttachFileWidgetPage` for APA but using `AttachFileWidgetCloudPage` which requires this method

Reply:
As only the design of `attach-file-cloud-widget.component` is changed mat-list remains the same in  `attach-file-widget.component` needed to update the attach-file cloud widget page

Reply:
Are you sure? We are using it here https://github.com/Alfresco/alfresco-apps/blob/develop/e2es/content-ee-apa/process-services-cloud-extension-e2e/src/tests/start-process/start-process-on-personal-files.e2e.ts#L133-L135
and in many other places. All the e2e related to processes are using it

Reply:
Yes, I have updated these files in the new PR that I'm going to create now in apps

Reply:
the methods they are using are from ADF, shouldn't you be updating the ADF methods? Are you creating new methods in apps?

Reply:
I haven't created any methods in apps, I have already updated the methods in the last ADF PR https://github.com/Alfresco/alfresco-ng2-components/pull/7324/files where I made changes for attach-file-widget, so only e2es passed for me when I merged new features of attach-file-widget. Just this method was missing in `AttachFileWidgetCloudPage` which is required in apps to check for no files.

Reply:
Guys, do we need a chat?

Reply:
You haven't updated all of them, I still see mat-list-item locators https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/testing/src/lib/protractor/core/pages/form/widgets/attach-file-widget.page.ts#L73

Reply:
yes let's continue this on a call

Reply:
yes, mat-list-item still exists for` attach-file-widget` component, only `attach-file-widget-cloud` component ui is changed